res/values/strings.xml

```diff
@@ -11354,4 +11354,6 @@
 
     <string name="deny_new_usb_title">USB accessories</string>
     <string name="deny_new_usb_summary">Control support for USB peripherals such as input (mice, keyboards, joysticks) and storage devices.</string>
+    <string name="scramble_pin_title">PIN Scrambling</string>
+    <string name="scramble_pin_summary">Controls PIN scrambling option when inputting PIN on Keyguard Pad.</string>
```

Comment:
This is user-facing text, perhaps something more familiar/intuitive other than "Keyguard" should be used to describe the screen lock PIN entry. I don't think an end user ever sees "Keyguard" being used as a string in the Settings app. The settings just refer to it as "Screen lock PIN", for example.